src/app/components/utility/modal/modal.component.ts

```diff
@@ -148,6 +189,36 @@ export class ModalComponent implements OnInit {
       } else {
         this.isDisabled = true;
       }
+    } else if (e.target.name === 'update_first_name') {
```

Comment:
Can you find better way (better than long `else if` conditions ) ? Also you can have separate method for checking each field, rather than having it in one. One method should do one thing.

Reply:
simplified as @lunayach request change on this